excelview.txt

```diff
@@ -6,7 +6,7 @@ Description = Freeware spreadsheet viewer for Office formats (*.xls).
 Size = 74.1 MiB
 Category = 6
 URLSite = http://support.microsoft.com/kb/180071/
-URLDownload = http://download.microsoft.com/download/e/a/9/ea913c8b-51a7-41b7-8697-9f0d0a7274aa/ExcelViewer.exe
+URLDownload = https://web.archive.org/web/20171225133033/http://download.microsoft.com/download/e/a/9/ea913c8b-51a7-41b7-8697-9f0d0a7274aa/ExcelViewer.exe
```

Comment:
Do these links work without the `if_` ?

Reply:
I'm not sure, these links are already in rapps-db repository (someone else updated them).